xbmc/Application.cpp

```diff
@@ -2737,6 +2737,7 @@ void CApplication::FrameMove(bool processEvents, bool processGUI)
       {
         CRenderingStats stats;
         g_windowManager.Process(CTimeUtils::GetFrameTime(), stats);
+        g_Windowing.SetVisible(stats.visible > 0);
```

Comment:
I would rather see pi polling this info from guilib than creating a new interface only a single platform will use.